zigbee: Fix setting intervals between image query (#7680)
nrfconnect/sdk-nrf: doc/nrf/libraries/zigbee/zigbee_fota.rst

OTA upgrade process
*******************

> OTA upgrade Client queries for OTA Upgrade servers with intrvals defined by :kconfig:option:`CONFIG_ZIGBEE_FOTA_SERVER_DISOVERY_INTERVAL_HRS` until at least one server is found.

Reviewer: intrvals -> intervals

Author: Fixed

> OTA upgrade Client queries for OTA Upgrade servers with intervals defined by :kconfig:option:`CONFIG_ZIGBEE_FOTA_SERVER_DISOVERY_INTERVAL_HRS` until at least one server is found.

Reviewer 2: ```suggestion
The OTA Upgrade Client queries for OTA Upgrade Servers with the intervals defined by the :kconfig:option:`CONFIG_ZIGBEE_FOTA_SERVER_DISOVERY_INTERVAL_HRS` Kconfig option until at least one server is found.
```

Author: Fixed.

> Once found, it will start to query the server for images. The interval between queries for the available Zigbee FOTA images is defined by :kconfig:option:`CONFIG_ZIGBEE_FOTA_IMAGE_QUERY_INTERVAL_MIN`.

Reviewer 2: ```suggestion
Once found, the client starts to query the server for images. 
The interval between queries for the available Zigbee FOTA images is defined by the :kconfig:option:`CONFIG_ZIGBEE_FOTA_IMAGE_QUERY_INTERVAL_MIN` Kconfig option.
```

Author: Fixed.
